app/screens/home/recent_mentions/index.ts

```diff
@@ -17,10 +19,29 @@ import {getTimezone} from '@utils/user';
 import RecentMentionsScreen from './recent_mentions';
 
 import type {WithDatabaseArgs} from '@typings/database/database';
+import type ServersModel from '@typings/database/models/app/servers';
 
 const enhance = withObservables([], ({database}: WithDatabaseArgs) => {
     const currentUser = observeCurrentUser(database);
 
+    // Get app database to query active servers
+    const appDatabase = DatabaseManager.appDatabase?.database;
+
+    // Create an observable to check for multiple workspaces
+    const hasMultipleTeams: Observable<boolean> = appDatabase ?
+        appDatabase.get<ServersModel>(MM_TABLES.APP.SERVERS).
+            query(
+                Q.and(
+                    Q.where('identifier', Q.notEq('')),
+                    Q.where('last_active_at', Q.gt(0)),
+                ),
+            ).
+            observe().
+            pipe(
+                map((servers: ServersModel[]) => servers.length > 1),
+            ) :
+        of$(false);
```

Comment:
this is wrong, you need to check for teams not servers.

in this case you could use `queryJoinedTeams(database).observeCount()` and then pipe it to get your boolean value.